group-coordinator/src/main/java/org/apache/kafka/coordinator/group/assignor/GroupSpecImpl.java

```diff
@@ -35,6 +37,11 @@ public class GroupSpecImpl implements GroupSpec {
      */
     private final SubscriptionType subscriptionType;
 
+    /**
+     * Partitions assigned keyed by topicId.
+     */
+    private final Map<String, Map<Uuid, Set<Integer>>> assignedPartitions;
```

Comment:
Is this the targetAssignment? If so, should we call it targetAssignment to be aligned with invertedTargetAssignment?

Reply:
In my head, target means the future assignment that the member must achieve right? And this is what it currently has

Reply:
This is correct. So why did you use `invertedTargetAssignment`? This is inconsistent with your thoughts. So, I think that we should either use targetAssignment for both or another name for both. We could perhaps use  lastTargetAssignment and invertedLastTargetAssignment if you don't want to use targetAssignment.

Reply:
I missed that, my bad. Yes I like that, I'll rename them